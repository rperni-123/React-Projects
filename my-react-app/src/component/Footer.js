import react,{Fragment} from 'react';

const Footer = (props) => {
    
    return(
        <Fragment>
            <div style={{position: 'fixed', left: '0', bottom: '0', width: '100%'}}>
            <hr/>
            <center>
                <h3>&copy;  Developer's Funnel {props.year} {props.month}</h3>
            </center>
            </div>
            
        </Fragment>
    )
}
export default Footer;
