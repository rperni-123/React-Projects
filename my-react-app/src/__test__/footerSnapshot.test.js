import React from 'react';
import Footer from '../component/Header';
import {create} from 'react-test-renderer'

describe('snapshot testing', () => {
    test('testing footer', () => {
        let tree = create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})

