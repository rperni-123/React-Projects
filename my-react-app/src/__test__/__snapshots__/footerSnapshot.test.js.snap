// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing testing footer 1`] = `
<header>
  <div
    className="logo"
  >
    Edureka
  </div>
  <center>
    <input
      onChange={[Function]}
    />
    <div
      id="test"
      style={
        Object {
          "color": "white",
          "fontSize": "15px",
        }
      }
    >
      user input here
    </div>
  </center>
  <hr />
</header>
`;
